chrome/content/zotero/integration/quickFormat.js

```diff
@@ -1225,27 +1345,29 @@ var Zotero_QuickFormat = new function () {
 	function _showCitationProperties(target) {
```

Comment:
I guess the source of the confusion of the item popover name is in the code itself, so probably should rename this function along with other renames.